AntennaTracker/GCS_MAVLink_Tracker.cpp

```diff
@@ -6,7 +6,7 @@ MAV_TYPE GCS_Tracker::frame_type() const
     return MAV_TYPE_ANTENNA_TRACKER;
 }
 
-MAV_MODE GCS_MAVLINK_Tracker::base_mode() const
+MAV_MODE_FLAG GCS_MAVLINK_Tracker::base_mode() const
```

Comment:
This doesn't look right.

The return value from this will often not be an entry in that enumeration. ... `MAV_MODE` also isn't correct for the same reason.

IMO we are going to change this we should move to `uint8`.